### ISSUE TITLE
Sign up does ask for an email

### ISSUE THREAD

**Issue by teinett** [pepyatka/pepyatka-html#116]
At this moment sign up requires just two fields to be filled in: username and password. It should require email as well.

**Comment by epicmonkey** [comment]
I'm going to keep sign up process as simple as possible before we finalise and speak out all pros and cons of an extended registration, i.e. there are two main reasons applications keep email in their sign up forms: a) they want to send email notifications to user once in a blue moon b) this is (kind of) security measure to protect applications from unapproved accounts. However some of web applications do not require email and still work brilliantly: reddit.com and news.ycombinator.com. Let's discuss.

**Comment by smallq-git** [comment]
@epicmonkey Note that your examples are not social networks or social networks for tech-savvy people. Facebook, Twitter, FriendFeed all require email. It's used for both notifications and tech support like password restoration. Since it's expected that users are going to lose and forget their passwords, we need to request their emails.

That aside, we perform username matching by email, and also there were plans to notify by email when new release is ready. 

To sum this up, we need to require email (and verify it!) during registration.
